Tests/SwerverTests/ResponseSpec/ResponseBuilderSpec.swift

```diff
@@ -0,0 +1,47 @@
+import Quick
+import Nimble
+import Swerver
+
+class ResponseBuilderSpec: QuickSpec {
+    override func spec() {
+        describe("A ResponseBuilder") {
+            var responseBuilder: ResponseBuilder!
+            
+            beforeEach {
+                responseBuilder = ResponseBuilder()
+            }
+            
+            it("it builds a response object that has a data value") {
+                let responseToBeTested = responseBuilder.build()
+                let expectedDate: String = responseToBeTested.getHeaderComponents()["Date"]!
+                
+                expect(expectedDate).toNot(beNil())
+            }
+            
+            it("can build a response with a status code value and a date value") {
```

Comment:
`it("can build` would be better worded as `it("builds`

In general, avoid things like "can" or "should" in test descriptions.